job-search-web-scraping.py

```diff
@@ -31,14 +33,19 @@ def send_email(sender_email_address, email_password,
     message['From'] = sender_email_address
     message['To'] = receiver_email_address
 
-    # attach the text file
+    # attach text in email body
+    text = MIMEText(open(file_path).read(), "plain")
+    message.attach(text)
     message.attach(MIMEText(email_body))
+    # attach the text file
     part = MIMEApplication(open(file_path).read())
     part.add_header('Content-Disposition', 'attachment; filename="%s"' % basename(file_path))
     message.attach(part)
-
-    # set smtp server and port
+    # set smtp server and port (SUGGEST TO ENABLE THE PORT VIA CONFIG (PARAMETER))
     server = smtplib.SMTP(email_smtp, '587')
+
+    # server = smtplib.SMTP_SSL(email_smtp, '587') THIS OTHER WAY TO SEND SECURE EMAILS COMMENTING LINES 50-52-53
```

Comment:
@feriodismo I checked this suggestion but didn't work for me. Does it work for you? 
I get the error:
```
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)
```

Reply:
I got this error when I use port 587:

`
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1091)

`

If I use port 465 I got no error. ( I am using "smtp.gmail.com" )

Reply:
Are you still getting the error changing the port?

Reply:
@feriodismo It still doesn't work if I use port 465. If I do so the program will hang at the line:
```
server = smtplib.SMTP_SSL(email_smtp, '465')
```
It seems we are using different versions of the ssl module...
Log from my module:
```
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)
```
Versus yours:
```
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1091)
```
What's your python version? Mine is Python 3.8.5.

Reply:
Maybe is that, I'm using Python 3.7.8 on windows

Reply:
@feriodismo Okay, i'll remove that suggestion from the code and merge. Will need to be sure others can run it to include it in the code.
Other than that emailing works fine. Thanks for your contribution!